numpyro/infer/initialization.py

```diff
@@ -25,6 +25,12 @@ def init_to_median(site=None, num_samples=15):
         and not site["is_observed"]
         and not site["fn"].is_discrete
     ):
+        if site["value"] is not None:
+            warnings.warn(
+                f"Site '{site['name']}' already stores a value - skip initializing this site."
```

Comment:
nit: I think it would be clearer to warn

> init_to_median() skipping initialization of site {} which already stores a value.

since 'skip' could be interpreted as a command (hey you should skip this) rather than a statement (hey I'm skipping this).

Reply:
Thanks, I have made the change.